spec/fixtures/actor.rb

```diff
@@ -1,12 +1,14 @@
 require 'active_support/cache'
 
 class Actor < User
-  attr_accessor :movies, :movie_ids
+  attr_accessor :movies, :movie_ids, :age, :birthplace, :show_birthplace
```

Comment:
I don't think we need `:show_birthplace` in the list of attributes, let's just keep it as a method... What do you think?